public/practice2/practice2.css

```diff
@@ -0,0 +1,37 @@
+.container{
```

Comment:
ここはスペース欲しいです！
他にも見受けられるので確認お願いします！
```suggestion
.container {
```